src/tuples.py

```diff
@@ -35,10 +35,13 @@ def dist(a, b):
 # Write a function `print_tuple` that prints all the values in a tuple
 
 # YOUR CODE HERE
-
+def print_tuple(n):
+  for item in n:
+    print(item)
+  
```

Comment:
Instead of adding the comma to make the argument a tuple (which is clever), you could look to see if the type of the argument is an int, and if it is, just print it. Then have an else which uses the for loop logic you have here.